<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>autoupdate_dialog</class>
 <widget class="QDialog" name="autoupdate_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Auto-update</string>
  </property>
  <layout class="QGridLayout">
   <item row="1" column="0">
    <widget class="QTextBrowser" name="textBrowser">
     <property name="openExternalLinks">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="autoinstall">
     <property name="text">
      <string>Automatically shutdown when download finishes</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="shutdown_and_update_button">
       <property name="text">
        <string>&amp;Shutdown &amp;&amp; Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="download_button">
       <property name="text">
        <string>&amp;Download</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="status_label">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>autoinstall</tabstop>
  <tabstop>shutdown_and_update_button</tabstop>
  <tabstop>download_button</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>textBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
